upup/pkg/fi/cloudup/gcetasks/address.go

```diff
@@ -130,9 +140,15 @@ func (_ *Address) CheckChanges(a, e, changes *Address) error {
 func (_ *Address) RenderGCE(t *gce.GCEAPITarget, a, e, changes *Address) error {
 	cloud := t.Cloud
 	addr := &compute.Address{
-		Name:    *e.Name,
-		Address: fi.ValueOf(e.IPAddress),
-		Region:  cloud.Region(),
+		Name:        *e.Name,
+		Address:     fi.ValueOf(e.IPAddress),
+		AddressType: fi.ValueOf(e.IPAddressType),
+		Purpose:     "SHARED_LOADBALANCER_VIP",
```

Comment:
Do we need to specify this?

Reply:
We need it, because we have it shared between `kops-controller` and `api-server`.